meetings/index.md

```diff
@@ -7,7 +7,7 @@ permalink: /meetings
 
 Monthly PMIx Standard Meeting
 -----------------------------
- - Second Friday of the month at 11am US Eastern / 8am US Pacific
+ - [Monthly PMIx Standard Meeting](https://github.com/pmix/pmix-standard/wiki#regular-teleconference-meeting-notes)
```

Comment:
Adjust the link slightly so that it points to the teleconf/ical portion of the wiki.
```suggestion
 - [Monthly PMIx Standard Meeting](https://github.com/pmix/pmix-standard/wiki#meeting-information)
```